### PR TITLE
All Prods. Listing Component

### LINKED ISSUE #183: All Products Listing
Visual Styles needed:
- Default (not a visual style)
- no-padding-top|No padding top|Remove top padding from this assembly
- no-padding-bottom|No padding-bottom|Remove bottom padding from this assembly
- dark|Dark|Dark background with white text
- left|Left|Left aligned content
- Gray BG

### PR DESCRIPTION
**Pull request by adelasofia**
All Prods. Listing Component with its visual styles `dark`, `gray`, `no-padding-bottom`, and `no-padding-top`.

Desktop (3 cols):
image

Tablet (2 cols):
image

Mobile (1 col.):
image

Closes #183

### CODE REVIEW

File: src/styles/rhd-theme/components/_all-products-listing.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.rhd-c-all-products-list {
+  @extend .pf-u-pt-lg;
+  @extend .pf-u-pb-lg;
+
+  h2, h2.pf-c-title {
+    margin-bottom: var(--pf-global--spacer--lg); // 24px
+  }
+
+  ul {
+    @extend .pf-l-grid;
```
Comment (mindreeper2420) [inline src/styles/rhd-theme/components/_all-products-listing.scss]: I would extend `.pf-c-list`, in order to utilize the PatternFly list styles, while still keeping your `list-style: none;` property.

**Comment by mindreeper2420** [comment]
To be consumed in the work for Issue https://github.com/redhat-developer/rhd-frontend/issues/212

**Comment by adelasofia** [comment]
@mindreeper2420 I fixed both issues 👍

**Comment by mindreeper2420** [comment]
Thanks @adelasofia ! Approved and merged.

**Pull request merged**